### PR TITLE
add golang upgarde cases check

### PR DESCRIPTION
**Pull request by zhaozhanqi**
before sdn migration only including cucushift cases, now adding golang cases as well 
cc @weliang1

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-sdn-live-migration-ovn-f28

**Comment by weliang1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-sdn-live-migration-ovn-f28

**Comment by weliang1** [comment]
@zhaozhanqi The testing failed on below case,
```
  failingScenarios:
    - Check sdn2ovn egressip is functional post upgrade - prepare
```
Will the @qeci tag from https://github.com/weliang1/verification-tests/blob/master/features/upgrade/sdn/sdn2ovn-migration.feature#L112 and https://github.com/weliang1/verification-tests/blob/8f61ab419e034ccb40513f7943339dd74ffa792c/features/upgrade/sdn/sdn2ovn-migration.feature#L43 cause this failure?

**Comment by zhaozhanqi** [comment]
> @zhaozhanqi The testing failed on below case,
> 
> ```
>   failingScenarios:
>     - Check sdn2ovn egressip is functional post upgrade - prepare
> ```
> 
> Will the @QeCi tag from https://github.com/weliang1/verification-tests/blob/master/features/upgrade/sdn/sdn2ovn-migration.feature#L112 and https://github.com/weliang1/verification-tests/blob/8f61ab419e034ccb40513f7943339dd74ffa792c/features/upgrade/sdn/sdn2ovn-migration.feature#L43 cause this failure?

cases failed is not related to tag,  please check the detailed logs.

**Comment by zhaozhanqi** [comment]
egressip failed was fix in https://github.com/openshift/openshift-tests-private/pull/18337

so this PR should be good to merged. @liangxia

File: ci-operator/step-registry/cucushift/sdn-live-migration-ovn/cucushift-sdn-live-migration-ovn-chain.yaml [hunk 1]
```diff
@@ -3,9 +3,12 @@ chain:
...
   - ref: cucushift-upgrade-prepare
+  - ref: cucushift-upgrade-prehealthcheck
   - ref: cucushift-sdn-live-migration-ovn
+  - ref: openshift-extended-upgrade-post
   - ref: cucushift-upgrade-check
   - ref: openshift-upgrade-qe-test-report
   documentation: |-
-    The workflow executes cucushift sdn live migration to ovn
\ No newline at end of file
+    The workflow executes cucushift sdn live migration to ovn
```
Comment (liangxia) [inline ci-operator/step-registry/cucushift/sdn-live-migration-ovn/cucushift-sdn-live-migration-ovn-chain.yaml:14]: Shall we update the doc to reflect the change, too ?

Comment (zhaozhanqi) [reply ci-operator/step-registry/cucushift/sdn-live-migration-ovn/cucushift-sdn-live-migration-ovn-chain.yaml]: Yes,  updated, thanks

**Pull request merged**